index.html

```diff
@@ -159,11 +159,9 @@ <h2 class="section-heading text-white text-center">Impact Meets Technology</h2>
             projects with passionate volunteers looking to make a positive contribution.
           </p>
           <!-- Begin Mailchimp Signup Form -->
-          <link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
+          <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
```

Comment:
you should keep the horizontal-slim, this forces the form to be one line as it is now :) @tiphaineboyer